c7n/resources/appelb.py

```diff
@@ -240,7 +335,20 @@ def _healthcheck_protocol_mismatch(alb):
 
 @filters.register('target-group')
 class AppELBTargetGroupFilter(ValueFilter, AppELBTargetGroupFilterBase):
-    """
+    """Filter ELB by their target group
+
+    :example:
+
+        .. code-block: yaml
+
+            policies:
+              - name: appelb-target-group
+                resource: app-elb
+                filters:
+                  - type: target-group
+                    key: length([?Protocol=='HTTPS'])
```

Comment:
also rather than copy and paste a filter, use an attribute unique to target group.